docs/0.4/index.md

```diff
@@ -0,0 +1,71 @@
+# Overview
+
+Splinter is a privacy-focused platform for distributed applications that
+provides a blockchain-inspired networking environment for private communication
+and transactions between organizations.
+
+## Getting Started
+
+  * [About Splinter](https://github.com/Cargill/splinter/blob/master/README.md)
+
+  * [Why use Splinter?](concepts/why_splinter.md)
+
+  * [Planning a Splinter deployment](howto/planning_splinter_deployment.md)
+
+## Splinter Concepts
+
+  * [Splinter glossary](glossary/glossary.md): Definitions for _circuit_,
+    _scabbard_, _state delta export_, and more
+
+  * [About Canopy](concepts/about_canopy.md): Splinter's distributed-application
+    UI framework that dynamically loads saplings (application UI components)
+
+  * [About Biome](concepts/about_biome.md): User management, private key
+    management, and credential management
+
+## Example Applications
+
+  * [Gameroom](https://github.com/Cargill/splinter/blob/master/examples/gameroom/README.md):
+    Web application that allows you to set up private multi-party circuits and
+    play tic tac toe with securely shared state
+
+## For Developers
+
+Splinter is an open-source project hosted at
+[github.com/Cargill/splinter](https://github.com/Cargill/splinter).
+This project operates under the [Cargill Code of
+Conduct](https://github.com/Cargill/code-of-conduct/blob/master/code-of-conduct.md).
+
+  * [Splinter release
+    notes](https://github.com/Cargill/splinter/blob/master/RELEASE_NOTES.md)
+
+  * [Stable feature checklist](../community/stable_feature_checklist.md)
```

Comment:
Any of the ".." should remain in the doc above this one (similar to FAQ).

Reply:
Updated